add dynamic sampling to the nav (#24033)
DataDog/documentation: content/en/logs/error_tracking/dynamic_sampling.md

---
title: Error Tracking Dynamic Sampling

> description:  Learn about how Dynamic Sampling in Error Tracking can make sure that your volume isn't consumed all at once.

Reviewer: ```suggestion
description:  Learn how to use Dynamic Sampling in Error Tracking to ensure your volume isn't consumed all at once.
```
